### PR TITLE
Added namspace chapter

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Fixed tickets | none
| License | MIT

#### What's in this PR?

Main PR https://github.com/sulu/sulu/pull/3094

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@danrot ping

File: bundles/markup/index.rst [hunk 1]
```diff
@@ -76,3 +76,28 @@ When registering your service simple add the tag
 
     In combination with the plugin-system of the :doc:`../admin/ckeditor` you
     can easily provide a user-interface to manage your custom tag.
+
+Namespaces
+----------
+
+Namespaces will be used to find tags with a special behavior. The default
+namespace is ``sulu`` but you can register your own namespace by adding a new
```
Comment (danrot) [inline bundles/markup/index.rst]: ``sulu``, but you can register

File: bundles/markup/index.rst [hunk 2]
```diff
@@ -76,3 +76,28 @@ When registering your service simple add the tag
...
+service and register you ``TagInterface`` implementations with this new
```
Comment (danrot) [inline bundles/markup/index.rst]: `your`

**Comment by wachterjohannes** [comment]
@danrot fixed.

**Pull request merged**